README.md

```diff
@@ -0,0 +1,98 @@
+    {install_path}/openmme/src/s11/conf/s11.json
+    {install_path}/openmme/src/s6a/conf/*.json
+    update the Diameter Configuration File:
+        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+       
+- the following values shall be defined:
+   - Identity:  
+      - the diameter identity.
+      - default value: mme.localdomain
+   - Realm:
+          - the diameter realm (everything past the first period of Identity)
+          - default value: localdomain
+          
+  - Refer free diameter  [config link](http://www.freediameter.net/trac/wiki/Configuration) 
+
+### Build open mme components as follows:
+    cd {install_path}/openmme
+    make clean; make; make install
+
+### Generate certificates   
+The "Diameter Identity" is a fully qualified domain name that is used to access the the Diameter peer. The "Diameter Host" is everything up to the first period of the "Diameter Identity". The  "Diameter Realm" is everything after the first period of the "Diameter Identity".
+
+Execute the following command, using the "Identity" configured in        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+
+    cd {install_path}/openmme/target/conf
+    ./make_certs.sh <diameter_host> <diameter_realm>
+    For Example: ./make_certs.sh mme localdomain
+
+   ### Start MME modules
+     cd {install_path}/openmme/target/
+     ./run.sh
+
+## Known Issues
+- O3 optimization flag is disabled in s6a module Observed issues in attach procedure when -O3 optimization flag is enabled in s6a module.
+ - APN value hardcoded in openMME  APN value sent in InitialContextSetupRequest message is hardcoded in the  following file,
+   - **FILE**: src/s1ap/handlers/attach_icsreq.c
+   - **Function**: icsreq_processing()
+   - **Code**:
+     - strncpy(apn_name, " apn", sizeof(" apn"));
+     - strcat(apn_name, ".TestNetwork");
+- Message Authentication Code (MAC) needs to be disabled at the RAN side to support number of devices >10 and S1-MME transaction rate >10
+- Multiple eNB support is added, but not tested beyond 2 eNB connections.
+
+## Appendix A.
+Before starting openMME applications respective parameters should be  configured in the json files mentioned in section 2.b. For each of the application individual json file can be created, or a user can create a common json file for entire MME configuration(all interfaces), and copy contents of this file in all other jsons.
+	OpenMME applications are designed modularly so that with extended IPC support user can have an individual application running on separate hosts/VMs, to help in this situation separate configuration files are created for each of the MME application(s1ap-app, s6a-app, s11-app, mme-app)
+
+### Config Parameters
+- Following is a description of each section under json file.
+	[U] - Reserved/Unused. Can be skipped in the json.
+	[M] - Mandatory parameter.
+
+#### "mme" - Contains information relevant for mme-app & s1ap-app.
+  - egtp_default_port[U] : Default S11 port for MME's s11 end.
```

Comment:
Better to make a 3 column table - name, mandator/unused, notes (including defaults if any)

Reply:
Done.